### DISCUSSION TITLE
Sunbird Knowlg - Release 6.1.0 Kick start

### DISCUSSION THREAD

**Discussion by Priyakashi** [Sunbird-Knowlg/Community#190] [General Discussions]
Hi all,
Release timelines for Sunbird Knowlg R6.1.0

![image](https://github.com/Sunbird-Knowlg/Community/assets/106071204/07db01f7-4362-42a6-ba0a-7ce90bdc282f)

Please find the tickets taken up in this release: [Link](https://project-sunbird.atlassian.net/issues/?filter=12877)

@mohitga ,@vinukumar-vs ,@pramodkvarma, @vrayulu, @SanthoshVasabhaktula, @rhwarrier, @pallakartheekreddy,
@rahul-kankaria, @ChakshuGautam, @kiranharidas187, @pritamps, @rahul101001000 @coolbung
@surendrasinghs, @pallakartheekreddy, @krgauraw, @luckynvdu @reshmi-nair, @sharathkashyap, @gandham-santhosh, @kameshbhr, @AmiableAnil, @Jayaprakash8887, @anandp504,[PoornimaRajesh](https://github.com/PoornimaRajesh)
@surendrasinghs
Shikshalokam: @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL
Samagra: @Prerna789 @aayushkedia95
SunbirdCB: @pkranga
Avanti: @pritamps @karn-nahata
DC's: @SanthoshVasabhaktula @vrayulu @maheshkumargangula
Sunbird QA: @ashokreddy1208
@gandham-santhosh
@kameshbhr @mmanimegalai
@madhucr @alok-os

**Comment by jayesh9747** [comment]
@Priyakashi  is there any documentation related to the Assessment-API micro service ?

**Reply by pallakartheekreddy** [reply]
@jayesh9747 The assessment service is part of the Inquiry Building block. Kindly submit your query on the forum provided below
https://github.com/orgs/Sunbird-inQuiry/discussions

**Reply by jayesh9747** [reply]
@pallakartheekreddy  
what is difference in sunbird knowledge and sunbird inquiry ?
i want to contribute in this issue https://github.com/shikshalokam/ml-survey-service/issues/140 
so that I read all about the sunbird-ml-survey service but i am confused where is knowledge assessment service , is inquiry building block is part of knowledge building block ?

**Comment by Priyakashi** [comment]
Hi all,

Please find the sign-off for Knowlg BB R6.1.0. We have an open bug for which fix will be provided as hotfix [mentioned in the sign off template].

Environment: (Functionalities were tested in the API level during regression)
Testing completion %: Completed (100%)
Planned sign-off date: 19th April 2024
Sign off date:  29th April 2024
Building block sign-off template: [Link](https://docs.google.com/spreadsheets/d/18ulgOn7Q6wnrai-DOcl6ldVdc1_eHqVQio9Iytgwvrw/edit#gid=71284613)

@mohitga ,@vinukumar-vs ,@pramodkvarma, @vrayulu, @SanthoshVasabhaktula, @rhwarrier, @pallakartheekreddy,
@rahul-kankaria, @ChakshuGautam, @kiranharidas187, @pritamps, @rahul101001000 @coolbung
@surendrasinghs, @pallakartheekreddy, @krgauraw, @luckynvdu @reshmi-nair, @sharathkashyap, @gandham-santhosh, @kameshbhr, @AmiableAnil, @Jayaprakash8887, @anandp504,[PoornimaRajesh](https://github.com/PoornimaRajesh)
@surendrasinghs
Shikshalokam: @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL
Samagra: @Prerna789 @aayushkedia95
SunbirdCB: @pkranga
Avanti: @pritamps @karn-nahata
DC's: @SanthoshVasabhaktula @vrayulu @maheshkumargangula
Sunbird QA: @ashokreddy1208
@gandham-santhosh
@kameshbhr @mmanimegalai
@madhucr @alok-os
